### DISCUSSION TITLE
Request: Pls add Helium(HNT) to Ledger Live and Nano X

### DISCUSSION THREAD

**Discussion by bttfw** [LedgerHQ/ledger-live-desktop#4000] [Coin/Token request]
Pls add Helium(HNT) to Ledger Live and Nano X

https://www.helium.com/

Their is an Helium App for Nano S mybe talk to them to add it to the hole Family

thx

**Comment by kellybyrd** [comment]
The helium Ledger app has been submitted to the Ledger app already to Ledger:
https://github.com/LedgerHQ/app-helium. From what I understand, it works on Nano S and X

It needs approval, then we're just missing support in Ledger Live.

**Comment by kellybyrd** [comment]
Thanks for moving this to a discussion. What can we do to help? The HNT community on Discord is excited for this. How can we help Ledger add this support?

**Reply by ericjohncarlson** [reply]
Voting support for this important addition to Helium and Ledger Live!

**Reply by Drew4200** [reply]
How and where do I vote?

**Reply by CryptoRedneck** [reply]
Sharing my support for this implementation here!
